_sections/120-entities.md

```diff
@@ -89,6 +143,23 @@ Entities are declared in an `entity` element in the [`meta` block](./#metadata)
   - Consumers of submissions that create entities MUST interpret "1" or "true" as indications to create an entity and any other value as indication not to create an entity
 - MUST have a direct child `label` representing a human-readable label
 
+### Declaring that a form updates entities
+
+*Added in spec version 2023.1.0*
+
+Entity updates are declared in the `entity` element in the [`meta` block](./#metadata) of the form definition. For entity updates, the `entity` element:
+
+- MUST be a direct child of `meta` in the primary instance.
+- MUST have attribute `id` populated by a [RFC 4122 version 4 UUID](https://www.rfc-editor.org/rfc/rfc4122) representing an existing entity
+  - Consumers of submissions that update entities MUST fail if the `id` attribute does not contain a UUID or if `id` does not reference an existing entity
+- MUST have attribute `dataset` representing the target Dataset for entities updated by submissions of this form
+- MUST have a `update` attribute populated with a "1" or "true" if the entity should be updated
+  - Consumers of submissions that update entities MUST interpret "1" or "true" as indications to update an entity and any other value as indication not to update an entity
+- MUST have a `baseVersion` attribute that is populated with the version of the entity that the form had access to
+- MAY also have a bind to a create attribute as previously defined. In that case, the form designer is responsible for making sure that the id is correctly populated in each case and that the update and create conditions are non-overlapping.
+- MAY have a direct child label representing a human-readable label
+
+
 ### Identifying entity properties
 
 The `entities:saveto` [`bind` attribute](./#bind-attributes) declares that the form field specified by the `nodeset` attribute on the bind should be saved as an Entity Property. The attribute's value is the Entity Property's name and and has the following restrictions:
```

Comment:
A question that came up from issa was which properties get updated. 

Once you think about adding `saveto`s to the properties you want to update, it starts to comes together, but I think it would be good to explicitly say in the update case that you use `saveto`s to say which properties you want to update and you can actually introduce new entity properties in an update form.